scripts/logstash-run.sh

```diff
@@ -5,17 +5,27 @@ export PATH=$BUILD_DIR/gradle/bin:$PATH
 
 SERVICE_URL="http://integration:9200"
 
+if [[ "$SECURE_INTEGRATION" == "true" ]]; then
+  SERVICE_URL="https://integration:9200 -k -u admin:admin"
+fi
+
 wait_for_es() {
-  count=120
+  count=20
```

Comment:
Why change?

Reply:
Before the method was sleeping 1 sec and run for 120 times. Now i updated sleep to 20 seconds, hence we don't have to run for 120 times, i updated to 20 times and end after that.

Reply:
With "sleep 20" iterating 20 times seems a bit more waiting time(20*20 = 400 seconds). How about reducing count to 10 or 5?

Reply:
Ack. Will update iterator to 5. (if opesearch couldn't come up in 100 Seconds, we will stop waiting)